File: Algoritmos Traduzidos/Perl/Algoritmos_Filipe_Dias/103.pl
print "Insira o ano corrente:\n";
$anoa = <STDIN>;
print "Insira o ano de nascimento:\n";
$anon = <STDIN>;
if ($anon > $anoa)
{
	print "\nAno de nascimento invalido";
}
else
{
	$idade = $anoa-$anon;
	print "\nIdade: $idade";
}
print "\n";

File: Algoritmos Traduzidos/Ruby/Algoritmos_Filipe_Dias/103.rb
puts "Insira o ano corrente:"
anoa = gets
puts "Insira o ano de nascimento:"
anon = gets
if anon.to_i > anoa.to_i
  puts "Ano de nascimento invalido"
else
  idade = anoa.to_i-anon.to_i
  puts "Idade: #{idade}"
end
  puts "\n"